### PR TITLE
[dataloader] CIFAR100

### PR DESCRIPTION
**Pull request by KSTconsult**

### CODE REVIEW

File: samples/cifar100/main.cpp [hunk 1]
```diff
@@ -0,0 +1,146 @@
+#include <iostream>
+
+#include <deepworks/deepworks.hpp>
+
+#include "utility.hpp"
+
+namespace dw = deepworks;
+
+static dw::Model buildCIFAR100Model(int batch_size) {
+    int mid_features_first = 384;
```
Comment (TolyaTalamanov) [inline samples/cifar100/main.cpp:10]: Is it the network from CIFAR10 sample ?

**Review by TolyaTalamanov — APPROVED** [review]
Just wondering whether the CIFAR10 topology work on CIFAR100 dataset

File: samples/cifar100/main.cpp [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+    int mid_features_second = 192;
+    int out_features = 10;
```
Comment (TolyaTalamanov) [inline samples/cifar100/main.cpp:12]: I believe there should be 100

File: samples/cifar100/main.cpp (hunk 1 above)
Comment (KSTconsult) [reply samples/cifar100/main.cpp:10]: Yep

File: samples/cifar100/main.cpp (hunk 2 above)
Comment (KSTconsult) [reply samples/cifar100/main.cpp:12]: My fault, sorry

File: samples/cifar100/main.cpp (hunk 1 above)
Comment (TolyaTalamanov) [reply samples/cifar100/main.cpp:10]: Do you have any accuracy results ?

**Comment by TolyaTalamanov** [comment]
@l-bat Have a look, can we merge it and start integration to CI and after that changing model to Resnet ?

**Review by l-bat — APPROVED** [review]
Thank you!

**Pull request merged**